### PR TITLE
Cleaned up DS RTC example

### PR DESCRIPTION
**Pull request by zgoda**
* cleaned up comments
* removed unnecessary code
* cleaned up serial printout

### CODE REVIEW

File: samples/DS3232RTC_NTP_Setter/app/application.cpp [hunk 1]
```diff
@@ -33,34 +32,26 @@ void onNtpReceive(NtpClient& client, time_t timestamp) {
...
 	// Station - WiFi client
 	WifiStation.config(WIFI_SSID, WIFI_PWD); // Put you SSID and Password here
 	WifiStation.enable(true);
-	WifiEvents.onStationGotIP(gotIP);
-	WifiEvents.onStationDisconnect(connectFail);
 
 	// set timezone hourly difference to UTC
 	SystemClock.setTimeZone(2); // GMT+2
 
+	System.onReady(systemReady);
```
Comment (slaff) [inline samples/DS3232RTC_NTP_Setter/app/application.cpp]: `WifiEvents.onStationGotIP(gotIP);` cannot be replaced with `System.onReady(systemReady);`. System.onReady sets callback to be triggered when the ESP system is ready. And `ready` != `STA connection got IP`

Comment (zgoda) [inline samples/DS3232RTC_NTP_Setter/app/application.cpp]: Many examples use onReady to initiate networked operations. Are these examples all wrong?

Comment (slaff) [inline samples/DS3232RTC_NTP_Setter/app/application.cpp]: > Many examples use onReady to initiate networked operations. 

It depends on your definition of "initiate networked operations".  System.onReady callbacks are  triggered after the TCP stack is ready. So far that should be included in "initiate networked operations". But that is different than connecting to an AP and getting IP address from the DHCP.   

> Are these examples all wrong?

Without knowing which examples you mean I cannot say for sure. Please, post here one or two names of samples that you've been referring to.

File: samples/DS3232RTC_NTP_Setter/app/application.cpp [hunk 2]
```diff
@@ -33,34 +32,26 @@ void onNtpReceive(NtpClient& client, time_t timestamp) {
 	Serial.printf("Time synchronized: %s\n", SystemClock.getSystemTimeString().c_str());
 }
 
-void gotIP(IPAddress ip, IPAddress netmask, IPAddress gateway)
+void gotIP(IPAddress ip, IPAddress netmask, IPAddress gw)
```
Comment (slaff) [inline samples/DS3232RTC_NTP_Setter/app/application.cpp]: Please, don't shorten the name of the variable `gateway` to `gw`.

File: samples/DS3232RTC_NTP_Setter/app/application.cpp [hunk 3]
```diff
@@ -38,29 +37,21 @@ void gotIP(IPAddress ip, IPAddress netmask, IPAddress gateway)
...
-	debugf("I'm NOT CONNECTED!");
-}
-
 void init()
 {
 	Serial.begin(SERIAL_BAUD_RATE);
-	Serial.systemDebugOutput(true); // Allow debug print to serial
 	Serial.println("Sming DSRTC_NTP_SETTER started!");
-//Uncomment proper line, or set right pins by hand
 	Wire.pins(0, 2); //Change to your SCL - 0,SDA - 2 GPIO pin number
```
Comment (avr39-ripe) [inline samples/DS3232RTC_NTP_Setter/app/application.cpp]: I think we shoudn't use Wire.pins(0, 2) by default, it is already default value, so better to comment out Wire.pins occurrences with comments how to set non-default scl/sda pins.

File: samples/DS3232RTC_NTP_Setter/app/application.cpp [hunk 4]
```diff
@@ -38,29 +37,21 @@ void gotIP(IPAddress ip, IPAddress netmask, IPAddress gateway)
...
 	// Station - WiFi client
 	WifiStation.config(WIFI_SSID, WIFI_PWD); // Put you SSID and Password here
 	WifiStation.enable(true);
-	WifiEvents.onStationGotIP(gotIP);
-	WifiEvents.onStationDisconnect(connectFail);
 
 	// set timezone hourly difference to UTC
 	SystemClock.setTimeZone(2); // GMT+2
 
+	WifiEvents.onStationGotIP(gotIP);
```
Comment (slaff) [inline samples/DS3232RTC_NTP_Setter/app/application.cpp]: Please,  move `WifiEvents.onStationGotIP ...` to be directly after `WifiStation.enable(tr...`.

**Pull request merged**